code/app/src/main/java/com/cmput301w21t06/crowdfly/Views/NewTrialActivity.java

```diff
@@ -120,9 +120,6 @@ public void onClick(View view) {
         addButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-
```

Comment:
I think this could be removed if it's not important. Since it might lead to unnecessary merge conflicts for others.